src/components/common/Extensions/NotificationBanner/NotificationBanner.tsx

```diff
@@ -31,10 +33,13 @@ const NotificationBanner: FC<PropsWithChildren<NotificationBannerProps>> = ({
         {
           'bg-success-100 border-success-200': status === 'success',
           'bg-warning-100 border-warning-200': status === 'warning',
+          'bg-warning-100 border-warning-400': status === 'notVerified',
```

Comment:
I don't think `notVerfied` is the correct status name for a banner

Reply:
what name do you suggest?